tests/semgrep/scripts/python_portability.py

```
@@ -0,0 +1,240 @@
+import os
+import subprocess
+import tempfile
+from pathlib import Path
+from subprocess import run as run_process
+from tempfile import NamedTemporaryFile as named_file
+
+import subprocess_utils as utils
+from subprocess_utils import run_git_command, run_git_command_with_input as git_input, run_safe_command
+
+
+def run_git_command_with_input(payload, argv, options) -> None:
```

Ruff (0.16.3)

[warning] Missing docstring in public function

(D103)